server/router.js

```diff
@@ -11,7 +11,7 @@ const { auth } = require('./middlewares/auth.js');
 const router = express.Router();
 
 router.get('/events', auth, getEvents);
-router.get('/checkauth', checkAuth);
+router.get('/checkauth', auth, checkAuth);
```

Comment:
same thing here, this file was in the previous PR, I merged it to master before!

Reply:
done :+1: